pkgs/top-level/python-packages.nix

```diff
@@ -12673,6 +12694,21 @@ in {
     };
   };
 
+  husl = buildPythonPackage rec {
+    pname    = "husl";
+    name    = "${pname}-${version}";
+    version = "4.0.3";
+
+    # ImportError: No module named tests
```

Comment:
ok...and how come? Should the test runner be invoked differently, or are tests simply missing? Just stating there was an importerror is not sufficient.